service/src/main/resources/db/migration/V243__payment.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TYPE payment_status AS ENUM ('DRAFT', 'SENT');
+
+CREATE TABLE payment (
+    id uuid PRIMARY KEY NOT NULL DEFAULT ext.uuid_generate_v1mc(),
+    created timestamp with time zone NOT NULL DEFAULT now(),
+    updated timestamp with time zone NOT NULL DEFAULT now(),
+    unit_id uuid NOT NULL CONSTRAINT fk$unit REFERENCES daycare (id),
```

Comment:
Index?

Reply:
Added in #2582. I didn't realize that this syntax doesn't add an index.